src/components/Home.tsx

```diff
@@ -4,12 +4,13 @@ import { Button, Grid } from '@material-ui/core';
 import HomeButtonsMajor from './HomeButtonsMajor';
 import HomeButtonsMinor from './HomeButtonsMinor';
 import Search from './Search';
+import { THomeButton } from '../types';
 
-const renderButtons = (buttons, props) =>
-  buttons.map((button, index) => {
+const renderButtons = (buttons: THomeButton[], props: any) =>
+  buttons.map((button: THomeButton, index: number) => {
     return (
       <Grid item xs={4} key={button.text}>
-        <Grid container direction="column" align="center">
+        <Grid container direction="column" alignItems="center">
           <Button
             component={Link}
```

Comment:
typechecker does not like this line... looking into it

Reply:
(talking about `component={Link}`

Reply:
try:
```
const renderButtons = (buttons: THomeButton[], props: { className: string }) =>
  buttons.map((button: THomeButton, index: number) => (
    <Grid item xs={4} key={button.text}>
      <Grid container direction="column" alignItems="center">
        <Button
          component={() => <Link to={button.to} />}
          data-test={button.text}
          {...props}
        >
          {button.icon}
          {button.text}
        </Button>
      </Grid>
    </Grid>
  ));
```

Reply:
[fixed](https://github.com/CodeForBoulder/upswyng/pull/52/commits/7f73281dd333f65fc3fbbfd55e9a7cf09eb6327d)